manifest.json

```diff
@@ -683,6 +683,19 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm",
+              "dockerfile": "src/alpine/3.9/arm32v7",
+              "os": "linux",
+              "tags": {
+                "alpine-3.9-arm32v7-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
+              },
+              "variant": "v7"
+            }
+          ]
+        },
```

Comment:
Nit: Could you actually move this up above the 3.9 ARM64 entry?  That'll keep things consistent where ARM32 is listed before ARM64 for everything.